src/swap/v1_1/index.ts

```diff
@@ -62,32 +62,26 @@ async function signTxns({
 
 async function generateTxns({
   client,
-  pool,
+  quote,
   swapType,
-  assetIn,
-  assetOut,
   slippage,
   initiatorAddr
-}: {
-  client: Algodv2;
-  pool: V1PoolInfo;
-  swapType: SwapType;
-  assetIn: AssetWithIdAndAmount;
-  assetOut: AssetWithIdAndAmount;
-  slippage: number;
-  initiatorAddr: string;
-}): Promise<SignerTransaction[]> {
+}: GenerateV1_1SwapTxnsParams): Promise<SignerTransaction[]> {
+  const {pool, quote: swapQuote} = quote;
+
+  const {assetInID, assetOutID} = swapQuote;
+
```

Comment:
```suggestion
  const {pool, quote: swapQuote} = quote;
  const {assetInID, assetOutID} = swapQuote;
```


I believe if there are not a lot of constants in a file, or if there is no certain distinction between variables, we don't need extra spaces between them. I guess it makes sense for the developer who is writing it, but when another dev reads, it just looks arbitrary. Also for other files